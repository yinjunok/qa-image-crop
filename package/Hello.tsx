import * as React from 'react';
import s from 'styled-components';

const H = s.h1`
  color: red;
`
let n: string = '1';

export default () => (
  <H>Hello World</H>
);