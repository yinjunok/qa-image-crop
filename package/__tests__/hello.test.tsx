// Link.react.test.js
import React from 'react';
import Hello from '../Hello';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <Hello />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});