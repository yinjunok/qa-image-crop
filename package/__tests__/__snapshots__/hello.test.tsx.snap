// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
.c0 {
  color: red;
}

<h1
  className="c0"
>
  Hello World
</h1>
`;
